tests/test-update-install-flatpaks.c

```diff
@@ -363,6 +364,26 @@ autoinstall_flatpaks_files_override (GFile                   *updater_directory,
                                                    error);
 }
 
+static GStrv
+parse_ostree_refspecs (const gchar  *ostree_refs_stdout,
+                       GError      **error)
+{
+  g_auto(GStrv) ostree_refs_stdout_lines = g_strsplit (ostree_refs_stdout, "\n", -1);
+  g_auto(GStrv) parsed_out_ostree_refs = g_new0 (gchar *, g_strv_length (ostree_refs_stdout_lines));
+  GStrv ostree_refs_stdout_lines_iter = ostree_refs_stdout_lines;
+  guint i = 0;
+
+  for (; *ostree_refs_stdout_lines_iter; ++ostree_refs_stdout_lines_iter)
+    {
+      if (g_strcmp0 (*ostree_refs_stdout_lines_iter, "") == 0)
+        continue;
+
+      parsed_out_ostree_refs[i++] = g_strdup (*ostree_refs_stdout_lines_iter);
+    }
+
+  return g_steal_pointer (&parsed_out_ostree_refs);
```

Comment:
Nitpick: `drop_empty_lines()` could be made quite a bit more efficient by not copying to a second array, and just rearranging the result of `g_strsplit()` to drop the empty lines (especially since they’re probably only at the end), but it’s test code and nobody cares about performance, so don’t bother addressing this. Just FYI.